client/src/components/App.css

```diff
@@ -1,3 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&display=swap');
+* {
```

Comment:
Ash, I believe when you use the '*' selector, that will change ALL of our margins and paddings. not just your component - is that right? and if so, I am ok with this. I prefer to start with a margin and padding of 0

Reply:
@aaronfife Yeah, you are right. It will affect all the component's stylings. I put it intentionally, but I can happily remove it if you think it might have any conflict with your individual component's styles.

Reply:
no i like it. i actually was going to do the same for mine but noticed it affected everyone. but if everyone is aligned lets keep it. i think it makes things easier

Reply:
👍 👍 👍